tests/testWPAuth0Helpers.php

```diff
@@ -67,4 +69,33 @@ public function testThatPluginSettingsLinksAreCorrectWhenReady() {
 			$plugin_links
 		);
 	}
+
+	public function testThatQueryVarHookIsAdded() {
+		$expect_hooked = [
+			'wp_auth0_register_query_vars' => [
+				'priority'      => 10,
+				'accepted_args' => 1,
+			],
+		];
+		$this->assertHookedFunction( 'query_vars', $expect_hooked );
+	}
+
+	public function testThatAuth0QueryVarsArePresent() {
+		$vars = wp_auth0_register_query_vars( [] );
+		$this->assertCount( 6, $vars );
+		$this->assertContains( 'error', $vars );
+		$this->assertContains( 'error_description', $vars );
+		$this->assertContains( 'a0_action', $vars );
+		$this->assertContains( 'auth0', $vars );
+		$this->assertContains( 'state', $vars );
+		$this->assertContains( 'code', $vars );
+
+		$vars = wp_auth0_register_query_vars( [ '__test_var__' ] );
+		$this->assertCount( 7, $vars );
+	}
+
+	public function testThatQueryVarsAreAddedProperly() {
+		$vars = wp_auth0_register_query_vars( [ '__test_var__' ] );
+		$this->assertCount( 7, $vars );
```

Comment:
shouldn't this honor the name of the test with something like 

`		$this->assertContains( 'code', $vars );`

instead?

Reply:
@lbalmaceda - Good catch. Looks like I started pulling that test out from the one above and never finished it.